virtcontainers/virtcontainers_test.go

```diff
@@ -48,10 +48,8 @@ var testHyperstartTtySocket = ""
 func cleanUp() {
 	globalSandboxList.removeSandbox(testSandboxID)
 	store.DeleteAll()
-	for _, dir := range []string{testDir, defaultSharedDir} {
-		os.RemoveAll(dir)
-		os.MkdirAll(dir, store.DirMode)
-	}
+	os.RemoveAll(testDir)
```

Comment:
Why remove `defaultSharedDir`?

Reply:
It was declared in the hyperstart_agent.go file, and seems to point to the hyper sandbox location, so I figured it may be irrelevant with the removal of hyperstart: `var defaultSharedDir = /run/hyper/shared/sandboxes/", but I'm not entirely sure

Reply:
The CI passed, so I think you should be right.